File: samples/javascript/sample.js
var start = Date.now() / 1000;
var end = start + 1;
var test = {
 "returncode": 0,
 "stdout": "Javascript stdout",
 "stderr": "Javascript stderr",
 "start_time": start,
 "end_time": end,
 "message": "Command message from javascript",
 "unit": "ms",
 "value": "22",
 "javascript": true,
 "provided_args": process.argv.slice(2),
}

console.log(JSON.stringify(test));

File: samples/azure/sample.py
import logging
import json
import azure.functions as func
import random
import json
import time


def main(req: func.HttpRequest) -> func.HttpResponse:
    logging.info('Python HTTP trigger function processed a request.')
    start_time = time.time()
    value = random.randint(1, 200)
    str_value = str(value)
    end_time = time.time()
    body_data = {}

    try:
        req_body = req.get_json()
        body_data = req_body
    except ValueError:
        pass

    json_return = {
        "returncode": 0,
        "stdout": "StdOut from azure",
        "stderr": "StdErr from azure",
        "start_time": start_time,
        "end_time": end_time,
        "message": "Random Value from Azure: " + str_value,
        "unit": "ms",
        "value": str_value,
        "azure": True,
        "body_data": body_data,
    }

    return func.HttpResponse(json.dumps(json_return))
